packages/core/src/util/ng_dev_mode.ts

```diff
@@ -115,7 +115,7 @@ export function initNgDevMode(): boolean {
     if (typeof ngDevMode !== 'object') {
       ngDevModeResetPerfCounters();
     }
-    return !!ngDevMode;
+    return typeof ngDevMode !== 'undefined' && !ngDevMode;
```

Comment:
Shouldn't it be?
```suggestion
    return typeof ngDevMode !== 'undefined' && ngDevMode;
```

(before it was `!!ngDevMode`)